src/pyk/proof/reachability.py

```diff
@@ -784,7 +797,7 @@ def lines(self) -> list[str]:
             f'    terminal: {self.terminal}',
             f'    refuted: {self.refuted}',
             f'    bounded: {self.bounded}',
-            f'    execution time: {self.exec_time // 3600}h {(self.exec_time % 3600) // 60}m {self.exec_time % 60}s',
+            f'    execution time: {self.exec_time}',
```

Comment:
Should this be `self.formatted_exec_time()`?
```suggestion
            f'    execution time: {self.formatted_exec_time()}',
```

Reply:
This is a parameter for the `APRSummary` which I called `exec_time : str`, but I can call it `formatted_exec_time`, maybe it makes more sense :)